test/e2e/common/node/pods.go

```diff
@@ -22,6 +22,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"k8s.io/client-go/util/retry"
```

Comment:
nit: import grouping is wrong, this should be in the last group below

Reply:
weird, this is how Goland added it...